client-http/pom.xml

```diff
@@ -69,6 +69,16 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpmime</artifactId>
+      <version>4.3.4</version>
```

Comment:
It seems there's a version 4.5.1 which is closer to what we're using above... although I'm really confused about the versioning of this project.